resources/core_rob_truck/client/functions.lua

```diff
@@ -0,0 +1,185 @@
+										Citizen.Wait(600)
+										DeleteEntity(GetClosestObjectOfType(playerpos.x,playerpos.y,playerpos.z,10.0,GetHashKey('prop_cash_pile_02')))
+										Citizen.Wait(400)
+									end
+									goto done
+								end
+							end
+						end
+					end
+				end
+			end
+			if marker_id ~= nil then
+				TriggerEvent("CORE_MARKERMENU:StopMarker_c",marker_id)
+				marker_id = nil
+			end
+			::done::
+		end
+		Citizen.Wait(sleep)
+	end
+end)
+
+RegisterNetEvent("CORE_ROB_TRUCK:Notification_c")
+AddEventHandler("CORE_ROB_TRUCK:Notification_c", function(msg)
+	BeginTextCommandThefeedPost("STRING")
+	AddTextComponentSubstringPlayerName(msg)
+	EndTextCommandThefeedPostTicker(true, false)
+end)
+RegisterNetEvent("CORE_ROB_TRUCK:PoliceNotification_c")
+AddEventHandler("CORE_ROB_TRUCK:PoliceNotification_c", function(coords,msg)
+	BeginTextCommandThefeedPost("STRING")
+	AddTextComponentSubstringPlayerName(msg)
+	EndTextCommandThefeedPostTicker(true, false)
+
+	local blip = AddBlipForCoord(coords.x,coords.y,coords.z)
+	SetBlipSprite(blip,477)
+	SetBlipColour(blip,1)
+	SetBlipScale(blip,2.0)
+	BeginTextCommandSetBlipName("STRING")
+	AddTextComponentString("Truck Robbery")
+	EndTextCommandSetBlipName(blip)
+
+	Citizen.CreateThread(function()
+		local alpha = 255
+		local status = true
+		while true do
+			local sleep = 1
+			if status == true then alpha = alpha - 1 else alpha = alpha + 1 end
+			if alpha == 0 then status = false end
+			if alpha == 255 then status = true end
+			SetBlipAlpha(blip,alpha)
+			Citizen.Wait(sleep)
+		end
+	end)
+	Citizen.Wait(1000 * 60 * 5)
+
+	RemoveBlip(blip)
+end)
+
+RegisterNetEvent("core_rob_truck:hint")
+AddEventHandler("core_rob_truck:hint", function()
+	local CasinoHint = "^0[^5Terminal At Risk^0] The upper exterior level of the Casino. The terminal is located [^9REDACTED^0]"
+	local HawickBankHint = "^0[^5Terminal At Risk^0] The exterior of the Bank at Hawick Ave. & Meteor St. The terminal is located [^9REDACTED^0]"
+	local GreatOceanHint = "^0[^5Terminal At Risk^0] The exterior of the Bank at Great Ocean Highway. The terminal is located [^9REDACTED^0]"
+
+	TriggerEvent("chatMessage", '', {0,0,0}, "^0[^3Technician Report^0] Vulnerabilities found in terminals across the city. [^9REDACTED^0] Exposes active bank truck locations. [^9REDACTED^0]")
+	Wait(3000)
+	TriggerEvent("chatMessage", '', {0,0,0}, "^0[^3Risk Analysis^0] Security Software is outdated and contains a vulnerability that can be used by hackers to expose precise GPS locations of trucks.")
+	Wait(3000)
+	local chance = math.random()
+	if chance <= 0.35 then
+		TriggerEvent("chatMessage", '', {0,0,0}, CasinoHint)
+	elseif chance >= 0.65 then
+		TriggerEvent("chatMessage", '', {0,0,0}, HawickBankHint)
+	else
+		TriggerEvent("chatMessage", '', {0,0,0}, GreatOceanHint)
+	end
+	Wait(4500)
+	TriggerEvent("chatMessage", '', {0,0,0}, "^0[^2Hint^0] You'll definitely need a bag to store the laptop and something... that goes boom...")
```

Comment:
```suggestion
	TriggerEvent("chatMessage", '', {0,0,0}, "^0[^2Hint^0] You'll definitely need a bag to store the money and something... that goes boom...")
```
the bag is to store the money from the truck right??

Reply:
well no. They actually need a bag to even get started with the hack. But I can put both

Reply:
ITS FOR ROLEPLAY MINI

Reply:
Updated